src/modules.rs

```diff
@@ -76,6 +76,6 @@ fn module_file(
 
     match parser::Parser::default_submod_path(id, dir_path, codemap).result {
         Ok(parser::ModulePathSuccess { path, .. }) => path,
-        Err(_) => panic!("Couldn't find module {}", id),
+        Err(_) => unreachable!("Couldn't find module {}", id),
```

Comment:
This probably shouldn't be `unreachable` - we're not saying its impossible to get here, we're just not handling the case where we do